components/brave_wallet/browser/blockchain_registry_unittest.cc

```diff
@@ -356,7 +356,7 @@ TEST(BlockchainRegistryUnitTest, GetBuyTokens) {
       mojom::OnRampProvider::kWyre, mojom::kPolygonMainnetChainId,
       base::BindLambdaForTesting(
           [&](std::vector<mojom::BlockchainTokenPtr> token_list) {
-            EXPECT_EQ(token_list.size(), 0UL);
+            EXPECT_EQ(token_list.size(), 2.0);
```

Comment:
This works, but let's do `2UL` for consistency. `UL` means unsigned long, by the way.

Reply:
@onyb Thank you. Making changes